zerver/migrations/0219_userprofile_is_primary_admin.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-06 11:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+def set_oldest_user_primary_admin(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model('zerver', 'Realm')
+    UserProfile = apps.get_model('zerver', 'Realm')
+    for realm in Realm.objects.all():
+        realm_admins = UserProfile.objects.filter(realm=realm, is_realm_admin=True)
+        oldest_realm_admin = realm_admins.order_by('-date').first()
```

Comment:
This would be `order_by('-date_joined')`, instead of `date`, I'll correct it soon :)